Source/.editorconfig

```diff
@@ -0,0 +1,34 @@
+﻿root = true
```

Comment:
Could you pleaase move `.editorconfig` file to the folder `src/dotnet/` instead of `Source/`